integration-test/src/main/scala/com/cloudera/livy/test/framework/BaseIntegrationTestSuite.scala

```diff
@@ -97,17 +97,18 @@ abstract class BaseIntegrationTestSuite extends FunSuite with Matchers {
     }
   }
 
-  test("initialize test cluster") {
+  override def beforeAll() {
```

Comment:
I don't remember exact details; but I believe some specific types of errors caused things to be really hard to debug. IIRC beforeAll runs in a static initialization context, and if certain exceptions happen at that point you have no idea what's going on.

I think using `before { }` to initialize the cluster should be enough, right? Only the first invocation takes long, all others are pretty much no-ops. (If doing that also keep the existing "initialize" test with an empty body, just to keep test running times accurate-ish.)